src/components/ButtonReport.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@chakra-ui/react';
+
+export type ButtonReportProps = {
+  onClick: () => void;
+  text: string;
+  title: string;
+  disabled: boolean;
+};
+
+export function ButtonReport({
+  onClick,
+  text,
+  title,
+  disabled
+}: ButtonReportProps) {
+  return (
+    <Button
+      bg="blue.600"
+      color="white"
+      onClick={onClick}
+      title={title}
+      _hover={{ bg: 'blue.800' }}
+      disabled={disabled}
+      mt={5}
```

Comment:
also here

Reply:
hey @joshreisner  in regards to this :
also not related but i'm seeing some major side-scrolling on mobile on aa-intergroup.org/meetings, seems like we need a solution for meeting descriptions with long URLs in them. (i also don't see why they are being entered since we have URL fields, but that's a separate conversation)

I have found the  troublesome css and removed it - was a knock on effect from not using chakra components - when I cleaned up the css from the app I was able to identify in the main site - all resolved :)